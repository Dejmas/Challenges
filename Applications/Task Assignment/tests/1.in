10
16 9 13 15 19 5 17 8 12 5
13 12 6 1 11 8 2 16 14 8
13 5 8 12 16 2 7 2 1 10
16 8 18 8 14 9 5 10 16 16
15 20 19 12 1 10 11 2 17 4
10 10 20 9 13 15 11 20 8 3
1 16 10 10 15 16 18 11 5 5
7 11 5 17 14 17 6 4 10 15
20 19 4 19 20 16 6 10 7 5
12 19 20 13 9 7 8 18 17 5
