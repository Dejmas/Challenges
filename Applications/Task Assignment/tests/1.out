36
1 2
2 4
3 9
4 7
5 5
6 10
7 1
8 8
9 3
10 6
